[2020-02-02 22:00:26,700] WARN Client session timed out, have not heard from server in 6808ms for sessionid 0x1000101245f0007 (org.apache.zookeeper.ClientCnxn)
[2020-02-02 22:00:26,780] INFO Client session timed out, have not heard from server in 6808ms for sessionid 0x1000101245f0007, closing socket connection and attempting reconnect (org.apache.zookeeper.ClientCnxn)
[2020-02-02 22:00:28,192] INFO Opening socket connection to server localhost/0:0:0:0:0:0:0:1:2181. Will not attempt to authenticate using SASL (unknown error) (org.apache.zookeeper.ClientCnxn)
[2020-02-02 22:00:28,193] INFO Socket connection established, initiating session, client: /0:0:0:0:0:0:0:1:50489, server: localhost/0:0:0:0:0:0:0:1:2181 (org.apache.zookeeper.ClientCnxn)
[2020-02-02 22:00:28,202] WARN Unable to reconnect to ZooKeeper service, session 0x1000101245f0007 has expired (org.apache.zookeeper.ClientCnxn)
[2020-02-02 22:00:28,203] INFO Unable to reconnect to ZooKeeper service, session 0x1000101245f0007 has expired, closing socket connection (org.apache.zookeeper.ClientCnxn)
[2020-02-02 22:00:28,203] INFO EventThread shut down for session: 0x1000101245f0007 (org.apache.zookeeper.ClientCnxn)
[2020-02-02 22:00:28,204] INFO [ZooKeeperClient Kafka server] Session expired. (kafka.zookeeper.ZooKeeperClient)
[2020-02-02 22:00:28,207] INFO [ZooKeeperClient Kafka server] Initializing a new session to localhost:2181. (kafka.zookeeper.ZooKeeperClient)
[2020-02-02 22:00:28,207] INFO Initiating client connection, connectString=localhost:2181 sessionTimeout=6000 watcher=kafka.zookeeper.ZooKeeperClient$ZooKeeperClientWatcher$@28eaa59a (org.apache.zookeeper.ZooKeeper)
[2020-02-02 22:00:28,212] INFO jute.maxbuffer value is 4194304 Bytes (org.apache.zookeeper.ClientCnxnSocket)
[2020-02-02 22:00:28,212] INFO zookeeper.request.timeout value is 0. feature enabled= (org.apache.zookeeper.ClientCnxn)
[2020-02-02 22:00:28,218] INFO Opening socket connection to server localhost/127.0.0.1:2181. Will not attempt to authenticate using SASL (unknown error) (org.apache.zookeeper.ClientCnxn)
[2020-02-02 22:00:28,219] INFO Socket connection established, initiating session, client: /127.0.0.1:50490, server: localhost/127.0.0.1:2181 (org.apache.zookeeper.ClientCnxn)
[2020-02-02 22:00:28,225] INFO Session establishment complete on server localhost/127.0.0.1:2181, sessionid = 0x1000101245f0008, negotiated timeout = 6000 (org.apache.zookeeper.ClientCnxn)
[2020-02-02 22:00:28,244] INFO Creating /brokers/ids/2 (is it secure? false) (kafka.zk.KafkaZkClient)
[2020-02-02 22:00:28,251] INFO Stat of the created znode at /brokers/ids/2 is: 352,352,1580709628248,1580709628248,1,0,0,72058698454728712,190,0,352
 (kafka.zk.KafkaZkClient)
[2020-02-02 22:00:28,251] INFO Registered broker 2 at path /brokers/ids/2 with addresses: ArrayBuffer(EndPoint(10.1.7.190,9092,ListenerName(PLAINTEXT),PLAINTEXT)), czxid (broker epoch): 352 (kafka.zk.KafkaZkClient)
[2020-02-02 22:00:28,322] WARN [Controller id=2, targetBrokerId=2] Error connecting to node 10.1.7.190:9092 (id: 2 rack: null) (org.apache.kafka.clients.NetworkClient)
java.net.SocketException: Network is unreachable
	at sun.nio.ch.Net.connect0(Native Method)
	at sun.nio.ch.Net.connect(Net.java:454)
	at sun.nio.ch.Net.connect(Net.java:446)
	at sun.nio.ch.SocketChannelImpl.connect(SocketChannelImpl.java:648)
	at org.apache.kafka.common.network.Selector.doConnect(Selector.java:280)
	at org.apache.kafka.common.network.Selector.connect(Selector.java:258)
	at org.apache.kafka.clients.NetworkClient.initiateConnect(NetworkClient.java:957)
	at org.apache.kafka.clients.NetworkClient.ready(NetworkClient.java:289)
	at org.apache.kafka.clients.NetworkClientUtils.awaitReady(NetworkClientUtils.java:65)
	at kafka.controller.RequestSendThread.brokerReady(ControllerChannelManager.scala:296)
	at kafka.controller.RequestSendThread.doWork(ControllerChannelManager.scala:250)
	at kafka.utils.ShutdownableThread.run(ShutdownableThread.scala:96)
[2020-02-02 22:00:28,429] WARN [Controller id=2, targetBrokerId=2] Error connecting to node 10.1.7.190:9092 (id: 2 rack: null) (org.apache.kafka.clients.NetworkClient)
java.net.SocketException: Network is unreachable
	at sun.nio.ch.Net.connect0(Native Method)
	at sun.nio.ch.Net.connect(Net.java:454)
	at sun.nio.ch.Net.connect(Net.java:446)
	at sun.nio.ch.SocketChannelImpl.connect(SocketChannelImpl.java:648)
	at org.apache.kafka.common.network.Selector.doConnect(Selector.java:280)
	at org.apache.kafka.common.network.Selector.connect(Selector.java:258)
	at org.apache.kafka.clients.NetworkClient.initiateConnect(NetworkClient.java:957)
	at org.apache.kafka.clients.NetworkClient.ready(NetworkClient.java:289)
	at org.apache.kafka.clients.NetworkClientUtils.awaitReady(NetworkClientUtils.java:65)
	at kafka.controller.RequestSendThread.brokerReady(ControllerChannelManager.scala:296)
	at kafka.controller.RequestSendThread.doWork(ControllerChannelManager.scala:250)
	at kafka.utils.ShutdownableThread.run(ShutdownableThread.scala:96)
[2020-02-02 22:00:28,536] WARN [Controller id=2, targetBrokerId=2] Error connecting to node 10.1.7.190:9092 (id: 2 rack: null) (org.apache.kafka.clients.NetworkClient)
java.net.SocketException: Network is unreachable
	at sun.nio.ch.Net.connect0(Native Method)
	at sun.nio.ch.Net.connect(Net.java:454)
	at sun.nio.ch.Net.connect(Net.java:446)
	at sun.nio.ch.SocketChannelImpl.connect(SocketChannelImpl.java:648)
	at org.apache.kafka.common.network.Selector.doConnect(Selector.java:280)
	at org.apache.kafka.common.network.Selector.connect(Selector.java:258)
	at org.apache.kafka.clients.NetworkClient.initiateConnect(NetworkClient.java:957)
	at org.apache.kafka.clients.NetworkClient.ready(NetworkClient.java:289)
	at org.apache.kafka.clients.NetworkClientUtils.awaitReady(NetworkClientUtils.java:65)
	at kafka.controller.RequestSendThread.brokerReady(ControllerChannelManager.scala:296)
	at kafka.controller.RequestSendThread.doWork(ControllerChannelManager.scala:250)
	at kafka.utils.ShutdownableThread.run(ShutdownableThread.scala:96)
[2020-02-02 22:00:28,646] WARN [Controller id=2, targetBrokerId=2] Error connecting to node 10.1.7.190:9092 (id: 2 rack: null) (org.apache.kafka.clients.NetworkClient)
java.net.SocketException: Network is unreachable
	at sun.nio.ch.Net.connect0(Native Method)
	at sun.nio.ch.Net.connect(Net.java:454)
	at sun.nio.ch.Net.connect(Net.java:446)
	at sun.nio.ch.SocketChannelImpl.connect(SocketChannelImpl.java:648)
	at org.apache.kafka.common.network.Selector.doConnect(Selector.java:280)
	at org.apache.kafka.common.network.Selector.connect(Selector.java:258)
	at org.apache.kafka.clients.NetworkClient.initiateConnect(NetworkClient.java:957)
	at org.apache.kafka.clients.NetworkClient.ready(NetworkClient.java:289)
	at org.apache.kafka.clients.NetworkClientUtils.awaitReady(NetworkClientUtils.java:65)
	at kafka.controller.RequestSendThread.brokerReady(ControllerChannelManager.scala:296)
	at kafka.controller.RequestSendThread.doWork(ControllerChannelManager.scala:250)
	at kafka.utils.ShutdownableThread.run(ShutdownableThread.scala:96)
[2020-02-02 22:00:28,751] WARN [Controller id=2, targetBrokerId=2] Error connecting to node 10.1.7.190:9092 (id: 2 rack: null) (org.apache.kafka.clients.NetworkClient)
java.net.SocketException: Network is unreachable
	at sun.nio.ch.Net.connect0(Native Method)
	at sun.nio.ch.Net.connect(Net.java:454)
	at sun.nio.ch.Net.connect(Net.java:446)
	at sun.nio.ch.SocketChannelImpl.connect(SocketChannelImpl.java:648)
	at org.apache.kafka.common.network.Selector.doConnect(Selector.java:280)
	at org.apache.kafka.common.network.Selector.connect(Selector.java:258)
	at org.apache.kafka.clients.NetworkClient.initiateConnect(NetworkClient.java:957)
	at org.apache.kafka.clients.NetworkClient.ready(NetworkClient.java:289)
	at org.apache.kafka.clients.NetworkClientUtils.awaitReady(NetworkClientUtils.java:65)
	at kafka.controller.RequestSendThread.brokerReady(ControllerChannelManager.scala:296)
	at kafka.controller.RequestSendThread.doWork(ControllerChannelManager.scala:250)
	at kafka.utils.ShutdownableThread.run(ShutdownableThread.scala:96)
[2020-02-02 22:00:28,860] WARN [Controller id=2, targetBrokerId=2] Error connecting to node 10.1.7.190:9092 (id: 2 rack: null) (org.apache.kafka.clients.NetworkClient)
java.net.SocketException: Network is unreachable
	at sun.nio.ch.Net.connect0(Native Method)
	at sun.nio.ch.Net.connect(Net.java:454)
	at sun.nio.ch.Net.connect(Net.java:446)
	at sun.nio.ch.SocketChannelImpl.connect(SocketChannelImpl.java:648)
	at org.apache.kafka.common.network.Selector.doConnect(Selector.java:280)
	at org.apache.kafka.common.network.Selector.connect(Selector.java:258)
	at org.apache.kafka.clients.NetworkClient.initiateConnect(NetworkClient.java:957)
	at org.apache.kafka.clients.NetworkClient.ready(NetworkClient.java:289)
	at org.apache.kafka.clients.NetworkClientUtils.awaitReady(NetworkClientUtils.java:65)
	at kafka.controller.RequestSendThread.brokerReady(ControllerChannelManager.scala:296)
	at kafka.controller.RequestSendThread.doWork(ControllerChannelManager.scala:250)
	at kafka.utils.ShutdownableThread.run(ShutdownableThread.scala:96)
[2020-02-02 22:00:28,961] WARN [Controller id=2, targetBrokerId=2] Error connecting to node 10.1.7.190:9092 (id: 2 rack: null) (org.apache.kafka.clients.NetworkClient)
java.net.SocketException: Network is unreachable
	at sun.nio.ch.Net.connect0(Native Method)
	at sun.nio.ch.Net.connect(Net.java:454)
	at sun.nio.ch.Net.connect(Net.java:446)
	at sun.nio.ch.SocketChannelImpl.connect(SocketChannelImpl.java:648)
	at org.apache.kafka.common.network.Selector.doConnect(Selector.java:280)
	at org.apache.kafka.common.network.Selector.connect(Selector.java:258)
	at org.apache.kafka.clients.NetworkClient.initiateConnect(NetworkClient.java:957)
	at org.apache.kafka.clients.NetworkClient.ready(NetworkClient.java:289)
	at org.apache.kafka.clients.NetworkClientUtils.awaitReady(NetworkClientUtils.java:65)
	at kafka.controller.RequestSendThread.brokerReady(ControllerChannelManager.scala:296)
	at kafka.controller.RequestSendThread.doWork(ControllerChannelManager.scala:250)
	at kafka.utils.ShutdownableThread.run(ShutdownableThread.scala:96)
[2020-02-02 22:00:29,067] WARN [Controller id=2, targetBrokerId=2] Error connecting to node 10.1.7.190:9092 (id: 2 rack: null) (org.apache.kafka.clients.NetworkClient)
java.net.SocketException: Network is unreachable
	at sun.nio.ch.Net.connect0(Native Method)
	at sun.nio.ch.Net.connect(Net.java:454)
	at sun.nio.ch.Net.connect(Net.java:446)
	at sun.nio.ch.SocketChannelImpl.connect(SocketChannelImpl.java:648)
	at org.apache.kafka.common.network.Selector.doConnect(Selector.java:280)
	at org.apache.kafka.common.network.Selector.connect(Selector.java:258)
	at org.apache.kafka.clients.NetworkClient.initiateConnect(NetworkClient.java:957)
	at org.apache.kafka.clients.NetworkClient.ready(NetworkClient.java:289)
	at org.apache.kafka.clients.NetworkClientUtils.awaitReady(NetworkClientUtils.java:65)
	at kafka.controller.RequestSendThread.brokerReady(ControllerChannelManager.scala:296)
	at kafka.controller.RequestSendThread.doWork(ControllerChannelManager.scala:250)
	at kafka.utils.ShutdownableThread.run(ShutdownableThread.scala:96)
[2020-02-02 22:00:40,455] INFO [GroupMetadataManager brokerId=2] Removed 0 expired offsets in 20 milliseconds. (kafka.coordinator.group.GroupMetadataManager)
